### PR TITLE
rego-cpp

### PR DESCRIPTION
**Pull request by matajoh**
### Why the changes in this PR are needed?

rego-cpp is a new integration, and I was encouraged to submit a PR to add it to the ecosystem page.

### What are the changes in this PR?

The necessary MD files to add rego-cpp as a new integration to the ecosystem page

### Notes to assist PR review:

Please let me know if I've missed anything.

### Further comments:

Feel free to reach out to me if you want to know more about rego-cpp (https://microsoft.github.io/rego-cpp/)

### CODE REVIEW

**Review by anderseknert — DISMISSED** [review]
This is great! Looks like we could use a C++ section in the [Awesome OPA](https://github.com/StyraInc/awesome-opa) list as well. PRs welcome ;)

File: docs/website/content/integrations/regocpp.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: regocpp
+software:
+- regopy
+- regorust
+inventors:
+- Microsoft
```
Comment (charlieegan3) [inline docs/website/content/integrations/regocpp.md:7]:
```suggestion
- microsoft
```

Comment (charlieegan3) [reply docs/website/content/integrations/regocpp.md:7]: We have an exiting org for MSFT, but it'll need to be lower cased to match.

It'll also appear on this page: https://www.openpolicyagent.org/organizations/microsoft/

**Comment by charlieegan3** [comment]
Thanks for this @matajoh, looks like a cool project. Keep us posted on any talks or blogs too and we can link them. Would love to learn more about the use cases you have for this.

**Comment by matajoh** [comment]
> This is great! Looks like we could use a C++ section in the [Awesome OPA](https://github.com/StyraInc/awesome-opa) list as well. PRs welcome ;)

Will do!

**Review by ashutosh-narkar — APPROVED** [review]

**Pull request merged**